package.json

```diff
@@ -19,5 +19,8 @@
     "eslint-plugin-import": "2.29.1",
     "stylelint": "16.3.1",
     "stylelint-config-standard": "36.0.0"
+  },
+  "dependencies": {
+    "driver.js": "^1.3.1"
```

Comment:
Sorry I don't see where driver.js is being imported ?

Reply:
Thanks, that's unused.